net/node/transactionPool.go

```diff
@@ -206,7 +221,7 @@ func (this *TXNPool) cleanTransactionList(txns []*transaction.Transaction) error
 	if txnsNum != cleaned {
 		log.Info(fmt.Sprintf("The Transactions num Unmatched. Expect %d, got %d .\n", txnsNum, cleaned))
 	}
-	log.Debug(fmt.Sprintf("[cleanProcessTxnPool], Requested %d clean, %d transactions cleaned from localNode.TransPool and remains %d still in TxPool", txnsNum, cleaned, this.GetTransactionCount()))
+	log.Info(fmt.Sprintf("[cleanTransactionList],transaction %d Requested, %d cleaned, Remains %d in TxPool", txnsNum, cleaned, this.GetTransactionCount()))
```

Comment:
I think this is needed, this message could display how many transactions has been persisted in a block. there have no message now can display how many transactions processed.